Manifest.toml

```diff
@@ -96,9 +63,11 @@ uuid = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 deps = ["Distributed", "InteractiveUtils", "Logging", "Random"]
 uuid = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
-[[UUIDs]]
-deps = ["Random"]
-uuid = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
+[[TimerOutputs]]
+deps = ["Crayons", "Printf", "Test", "Unicode"]
+path = "/Users/sabae/.julia/dev/TimerOutputs"
```

Comment:
This is WIP, I presume.

Reply:
Correct; we're about to land https://github.com/KristofferC/TimerOutputs.jl/pull/50 after which I'll eliminate this `dev` business.  Good eye, but I guess I should expect that you of all people would be checking out the `Manifest` diff.